back-end/helpers/test-helpers/testIntentGenerator.js

```diff
@@ -0,0 +1,97 @@
+export default class testIntentGenerator{
+    async generateOneIntentModel() {
+        var intents = [];
```

Comment:
This applies to the entire file - is there any reason we initialize an empty list, then declare model/analyzedIntentX, then push model/analyzedIntentX into the list in sequence? Can we declare them in the list initially?

i.e.
```suggestion
        var intents = [
            { question: "Waluigi ...",
            children: {"The brothers ...": 1},
            total_children: 1,
            project_id: 1
            }
        ];
```

Reply:
you... are not wrong